src/main/resources/application.properties

```diff
@@ -19,6 +18,15 @@ cas.saml.tolerance=180000
 cas.send.renew=false
 
 # =========================================================================
+app.iam.request.form=https://www.hawaii.edu/bwiki/display/UHIAM/UH+Groupings+Request+Form
```

Comment:
We inject this URL from our message.properties file currently, so we can remove this line.

Reply:
done